### DISCUSSION TITLE
error

### DISCUSSION THREAD

**Discussion by droy1up** [utmapp/UTM#4101] [General]
getting this when trying to boot up the vm: 'qemu-aarch64-softmmu: -netdev vmnet-shared,id=net0: cannot create vmnet interface: conflict, sharing service is in use'. 

I am on a M1 macbook pro 13 inch
<img width="262" alt="Screenshot 2022-06-07 at 22 24 33" src="https://user-images.githubusercontent.com/95874049/172485346-f1f76a05-07a6-4d69-ae54-082781753a10.png">
 
any reasons to this and any fixes/workarounds to this?

**Comment by AngshmnMzmdr** [comment]
Check your network settings, make sure its not shared(in case you have multiple roms installed) make it bridged.

**Reply by shreeya728** [reply]
Please elaborate

**Reply by vedanta28** [reply]
> Please elaborate

@shreeya728 
I think I got the solution. Go to the VM settings you have created. Then check the Network Section and change Network Mode to Bridged.
<img width="793" alt="image" src="https://user-images.githubusercontent.com/74861233/181273441-117d5674-1839-4164-8a38-803bbfe38122.png">

>

**Reply by JemmyRAC** [reply]
ty

**Reply by GiantHardOfficial** [reply]
i tried it and it never worked 
![Screen Shot 2023-07-30 at 3 22 58 PM](https://github.com/utmapp/UTM/assets/140940319/432a9d12-c52b-476a-82c8-97b7b5d9f7fb)

**Reply by GiantHardOfficial** [reply]
in case if your wondering about the picture i showed.. well i didnt have google chrome installed and that was my personal vm

**Reply by GiantHardOfficial** [reply]
but i didnt try the mac address so ill try it

**Comment by shramee** [comment]
### QEMU exited from an error: qemu-aarch64-softmmu: -netdev vmnet-shared,id=net0: cannot create vmnet interface: unknown vmnet error

[Setting Network mode to Bridged](https://github.com/utmapp/UTM/discussions/4101#discussioncomment-3261377) works a treat, thanks @vedanta28!

**Comment by JMOXIE** [comment]
GOOD

**Comment by quanwei2006** [comment]
when I run "./uup_download_macos.sh", I got issues as follows:

07/10 21:52:53 [ERROR] CUID#7 - Download aborted. URI=https://github.com/uup-dump/converter/raw/073071a0003a755233c2fa74c7b6173cd7075ed7/convert.sh
Exception: [AbstractCommand.cc:856] errorCode=1 Failed to establish connection, cause: Connection refused

07/10 21:52:53 [ERROR] CUID#8 - Download aborted. URI=https://github.com/uup-dump/converter/raw/073071a0003a755233c2fa74c7b6173cd7075ed7/convert_ve_plugin

gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
685754|ERR |       0B/s|files/convert.sh
af6a82|ERR |       0B/s|files/convert_ve_plugin

i don't know how to fix it.

**Comment by quanwei2006** [comment]
when I run "./uup_download_macos.sh", I got issues as follows:

07/10 21:52:53 [ERROR] CUID#7 - Download aborted. URI=https://github.com/uup-dump/converter/raw/073071a0003a755233c2fa74c7b6173cd7075ed7/convert.sh
Exception: [AbstractCommand.cc:856] errorCode=1 Failed to establish a connection, cause: Connection refused

07/10 21:52:53 [ERROR] CUID#8 - Download aborted. URI=https://github.com/uup-dump/converter/raw/073071a0003a755233c2fa74c7b6173cd7075ed7/convert_ve_plugin

gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
685754|ERR |       0B/s|files/convert.sh
af6a82|ERR |       0B/s|files/convert_ve_plugin

i don't know how to fix it.

**Comment by wn2515** [comment]
So i am getting this error all of a sudden, when this used to always work for me.  I have always used bridged networking.

UTM v4.4.5(94)

**Comment by Gvorlv** [comment]
getting this when trying to boot up the vm: QEMU error: QEMU завершила работу с ошибкой: qemu-aarch64-softmmu: -drive if=none,media=cdrom,id=drive73699276-7C6A-4E47-9E88-DB63295C8E9E,file=/Users/vladimirorlovskij/Downloads/debian-12.5.0-arm64-netinst.iso,readonly=on: Failed to lock byte 100 
<img width="332" alt="Снимок экрана 2024-03-07 в 10 35 51" src="https://github.com/utmapp/UTM/assets/102881604/9b1bd09a-0119-493e-bb62-3bce1c4fbdd7">
